packages/components/src/components/popup/popup.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Body from './popup-body.jsx';
+import Footer from './popup-footer.jsx';
+import Header from './popup-header.jsx';
+import Modal from '../modal';
+
+const Popup = ({
+    active_tab_icon_color,
+    balance,
+    className,
+    close_icon_color,
+    header_backgound_color,
+    header_contents_color,
+    header_icon,
+    renderFooter,
+    renderHeader,
+    should_show_popup,
+    tab_icon_color,
+    tabs_detail,
+    title,
+    togglePopupModal,
+}) => {
+    const header = () => {
+        const render_header = typeof renderHeader === 'function' ? renderHeader() : null;
+        return (
+            <Header
+                header={render_header}
+                title={title}
+                balance={balance}
+                text_color={header_contents_color}
+                header_icon={header_icon}
```

Comment:
Do we still need to pass these if `render_header` is defined?

Reply:
I remove `render_header`, although `render_header` was for extra use for the future, not a substitution for those.